resources/web/wwi/nodes/WbElevationGrid.js

```diff
@@ -99,4 +104,38 @@ export default class WbElevationGrid extends WbGeometry {
 
     return !invalid;
   }
+
+  _sanitizeFields() {
+    const newTickness = resetIfNegative(this.thickness, 0.0)
+    if (newTickness)
+      this.thickness = newTickness;
+
+    const newXDimension = resetIfNegative(this.xDimension, 0)
+    if (newXDimension)
+      this.xDimension = newXDimension;
+
+    const newXSpacing = resetIfNonPositive(this.xSpacing, 1.0)
+    if (newXSpacing)
+      this.xSpacing = newXSpacing;
+
+    const newYDimension = resetIfNegative(this.yDimension, 0)
+    if (newYDimension)
+      this.yDimension = newYDimension;
+
+    const newYSpacing = resetIfNonPositive(this.ySpacing, 1.0);
+    if (newYSpacing)
+      this.ySpacing = newYSpacing;
+
+    this._checkHeight();
+
+    return !newTickness && !newXDimension && !newXSpacing && !newYDimension && !newYSpacing;
```

Comment:
That seems odd as the default values for `newXDimension` is 0 and !0 is true while the default value for newXSpacing is 1 and !1 is false. See [here](https://stackoverflow.com/a/11550718). So I suspect there might be bug hidden here. Instead, we should probably test this way:
```suggestion
    return newTickness === false && newXDimension === false && newXSpacing === false && newYDimension === false && newYSpacing === false;
```
The same issue may arise in other implementations of the `_sanitizeFields()` method.